### PR TITLE
Properly get and use the user's name

### LINKED ISSUE #368: Author field should be auto-populated by the user name by default

### PR DESCRIPTION
**Pull request by Kapin**
Change does two key things

1. Adds first name and last name to the signup form
2. Makes the author field on procedure default to owner's full name if it is blank or non-existent.
3. Fixes the "Hello username" field
4. Fixes translations

Closes #368, #365, #380, #381

### CODE REVIEW

**Comment by Trinovantes** [comment]
Either the author field needs to be nullable or the first and last name cannot be null

**Comment by Kapin** [comment]
@Trinovantes that's what I did

**Comment by Trinovantes** [comment]
Then do it on the frontend instead of the backend so the user doesn't initially see an empty text field then a few seconds later get filled when the async request finishes. Ask @Ommy where we store the user name

**Comment by Kapin** [comment]
Need #370 merged first to do that properly.

**Comment by Ommy** [comment]
@Kapin #370 is merged

File: src-backbone/app/js/models/procedure.js [hunk 1]
```diff
@@ -12,10 +12,12 @@ let Procedure = Backbone.Model.extend({
     urlRoot: '/api/procedures',
 
     defaults: function() {
+        let user = App().session.getUser();
```
Comment (Ommy) [inline src-backbone/app/js/models/procedure.js]: not a method

Comment (Kapin) [inline src-backbone/app/js/models/procedure.js]: I'm following what you did for the concepts view. I'm confused.

Comment (Ommy) [inline src-backbone/app/js/models/procedure.js]: https://github.com/SanaMobile/sana.protocol_builder/blob/master/src-backbone/app/js/views/common/loggedInRightNavbarView.js#L10

Comment (Kapin) [inline src-backbone/app/js/models/procedure.js]: I added that function to session.js. I can do it that way, but I wanted to abstract away the key-value store.

Comment (Ommy) [inline src-backbone/app/js/models/procedure.js]: Oh oops, my mistake, I didn't see that. Carry on

Comment (Kapin) [inline src-backbone/app/js/models/procedure.js]: I'll change yours though :smile:

**Comment by Trinovantes** [comment]
is the getUser method still consistent with @Ommy's refactor?

**Comment by Kapin** [comment]
@Trinovantes this should go in before that.

**Comment by Kapin** [comment]
@Trinovantes I'll wait till he merges

**Comment by Ommy** [comment]
@Kapin #387 merged

File: src-backbone/app/js/views/procedures/proceduresNavbarView.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 const EventKeys = require('utils/eventKeys');
+let App = require('utils/sanaAppInstance');
```
Comment (Ommy) [inline src-backbone/app/js/views/procedures/proceduresNavbarView.js]: const

Comment (Kapin) [inline src-backbone/app/js/views/procedures/proceduresNavbarView.js]: done

**Comment by Ommy** [comment]
+1

**Pull request merged**